ui.apps/src/main/content/jcr_root/apps/venia/components/commerce/productcarousel/v1/productcarousel/clientlibs/css/product-carousel.css

```diff
@@ -0,0 +1,113 @@
+
+
+
+.blank-placeholder {
```

Comment:
For all the CSS class names, shouldn't we better use BEM style and change to `__`? Question goes also to @dplaton & @herzog31

Reply:
Yes. We use `__` everywhere, but according to BEM this class should be called `placeholder__empty`

Reply:
Yes, we should be consistent with the naming of all CSS classes, so customers can re-use them.

Reply:
It would be good to place this placeholder css at a common place. It is declared in Product Teaser also. @dplaton , is there a common.css etc we have we can use ?